test/conversion_filter_test.cpp

```diff
@@ -138,6 +138,15 @@ BOOST_AUTO_TEST_CASE(test_color_to_rgb) {
     BOOST_TEST(*map.eight_bit_channels["test:r"] == testr, "value r in filter color_to_rgb should be " + std::to_string(testr) + " but is " + std::to_string(*map.eight_bit_channels["test:r"]));
     BOOST_TEST(*map.eight_bit_channels["test:g"] == testg, "value g in filter color_to_rgb should be " + std::to_string(testg) + " but is " + std::to_string(*map.eight_bit_channels["test:g"]));
     BOOST_TEST(*map.eight_bit_channels["test:b"] == testb, "value b in filter color_to_rgb should be " + std::to_string(testb) + " but is " + std::to_string(*map.eight_bit_channels["test:b"]));
+
+    in_channel = dmxfish::dmx::pixel(0.0, 0.0, 1.0);
+    fil.update();
+    testr = 85; // TODO shouldn't this be 255?
```

Comment:
For my idea yes it should be 255, but this was the basic idea of the hsiluminance color room. Now the white has the same intensity of 255 (in sum of r, g, b) as a red with a value of 255.

Reply:
Let's deal with this after coal.